test/common/api/os_sys_calls_test.cc

```diff
@@ -72,7 +72,7 @@ TEST(OsSyscallsTest, SetAlternateGetifaddrs) {
   auto& os_syscalls = Api::OsSysCallsSingleton::get();
   const bool pre_alternate_support = os_syscalls.supportsGetifaddrs();
   Api::InterfaceAddressVector interfaces{};
-#if defined(WIN32) || (defined(__ANDROID_API__) && __ANDROID_API__ < 24)
```

Comment:
issetalternategetifsddrs used any more?  If not, can we remove?

Reply:
Removed!

Reply:
No, I don't think so. This doesn't change any API or behavior in Envoy Mobile itself (just an implementation detail about how this behavior is implemented). And while this does change code in envoy/api, this is not part of Envoy's "API" which is the protos.

Reply:
Thanks for clarifying, yes I was referring to this breaking Envoy's API, but it's good to know that Envoy's programmatic interface isn't considered a stable API, only the protos.